### DISCUSSION TITLE
Gopher2600 Windows 7 x64

### DISCUSSION THREAD

**Discussion by ctsmith2** [JetSetIlly/Gopher2600#32] [Q&A] [answered]
Hello,

Are there any releases of Gopher2600 that work on Windows 7 x 64.
Right now when I click on, "gopher2600_windows_amd64.exe" (v0.32.1), 
nothing happens.

Chris

**Comment by JetSetIlly** [comment, marked as answer]
Hi,

There are no recent releases that support Windows 7. Unfortunately, I'm driven by the requirement of the Go compiler which now create binaries that target at a minimum, Windows 10 and Server 2016. The reason for this is because Microsoft no longer support Windows 7.

It looks like [v0.26.2b](https://github.com/JetSetIlly/Gopher2600/releases/tag/v0.26.2b) was the last version that likely supported Windows 7.

I appreciate that this is a nuisance but there's not a lot I can do for newer releases. I will have started using new language features that require more recent versions of the compiler and hence cannot be compiled for Windows 7.


v0.26.2b will be good enough to give you an idea whether Gopher2600 is suitable for your needs but bear in mind there have a been a lot of improvements since then.

Regards
Stephen

**Reply by ctsmith2** [reply]
This version does work. Thank you! :)
